sklearn/metrics/ranking.py

```diff
@@ -22,6 +22,7 @@
 import warnings
 import numpy as np
 from scipy.sparse import csr_matrix
+import pandas
```

Comment:
actually, you can not import pandas AT ALL. What you refer to is a test, and you can't really do that same thing here. You can check the class name, though, if you like.